applications/FluidDynamicsApplication/custom_utilities/acceleration_limitation_utilities.cpp

```diff
@@ -0,0 +1,86 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+
+// Application includes
+#include "acceleration_limitation_utilities.h"
+
+namespace Kratos
+{
+
+    /* Public functions *******************************************************/
+
+    /**
+     * @brief Iteration over all nodes in ModelPart to find and reduce excessive accelerations.
+     * Model part is specified in the constructor
+     */
+
+    void AccelerationLimitationUtilities::Execute() {
+
+        double dt = mrModelPart.GetProcessInfo().GetValue(DELTA_TIME);
+
+        ModelPart::NodesContainerType rNodes = mrModelPart.Nodes();
+        #pragma omp parallel for
+        for(int count = 0; count < static_cast<int>(rNodes.size()); count++)
+        {
+            ModelPart::NodesContainerType::iterator i = rNodes.begin() + count;
```

Comment:
it is shorter and sometimes you can save an include this way
You don't have to use it if you are not comftable with it
Sometime it can screw you up :)